e2e/atlas/private_endpoint_test.go

```diff
@@ -317,32 +317,16 @@ func TestPrivateEndpointsAzure(t *testing.T) {
 
 var regionsGCP = []string{
 	"CENTRAL_US",
-	"EASTERN_US",
 	"US_EAST_4",
 	"NORTH_AMERICA_NORTHEAST_1",
-	"NORTH_AMERICA_NORTHEAST_2",
 	"SOUTH_AMERICA_EAST_1",
 	"WESTERN_US",
 	"US_WEST_2",
-	"US_WEST_3",
-	"US_WEST_4",
-	"EASTERN_ASIA_PACIFIC",
-	"ASIA_EAST_2",
-	"NORTHEASTERN_ASIA_PACIFIC",
-	"ASIA_NORTHEAST_2",
-	"ASIA_NORTHEAST_3",
-	"SOUTHEASTERN_ASIA_PACIFIC",
-	"ASIA_SOUTH_1",
-	"ASIA_SOUTH_2",
-	"AUSTRALIA_SOUTHEAST_1",
 	"AUSTRALIA_SOUTHEAST_2",
 	"ASIA_SOUTHEAST_2",
 	"WESTERN_EUROPE",
 	"EUROPE_NORTH_1",
 	"EUROPE_WEST_2",
-	"EUROPE_WEST_3",
-	"EUROPE_WEST_4",
-	"EUROPE_WEST_6",
 	"EUROPE_CENTRAL_2",
 }
```

Comment:
atlas team suggested using max 12 regions for GCP private endpoints as we may hit internal limit

Reply:
got it! thanks for looking into this!